theme/ckeditor5-ui/components/icon.css

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * Default width/height of an icon. Note that it equals the default
+ * height of the line of text, so e.g. it matches the height of the
+ * button's label.
+ */
+:root {
+	--ck-icon-min-size: calc(var(--ck-font-size-base) * var(--ck-line-height-base));
+}
+
+svg.ck-icon {
+	width: calc(var(--ck-line-height-base) * 1em);
+	height: calc(var(--ck-line-height-base) * 1em);
```

Comment:
I see `--ck-font-size-normal: 1em`, is it worthy to change it?

Reply:
What do you mean?

Reply:
I mean 
```css
width: calc(var(--ck-line-height-base) * var(--ck-font-size-normal));
```